package.json

```diff
@@ -60,54 +60,45 @@
   },
   "dependencies": {
     "@data-exp-lab/yt-tools": "^0.4.1",
-    "@jupyter-widgets/base": "^1.1.10 || ^2 || ^3 || ^4",
-    "@jupyterlab/application": "^3.0.4",
-    "@jupyterlab/coreutils": "^5.0.2",
-    "@jupyterlab/mainmenu": "^3.0.3",
-    "@jupyterlab/services": "^6.0.3",
+    "@jupyter-widgets/base": "^1.1.10 || ^2 || ^3 || ^4 || ^5 || ^6",
+    "@jupyterlab/application": "^4.2.5",
     "@types/fscreen": "^1.0.1",
     "@types/node": "^10.11.6",
     "@types/three": "^0.141.0",
     "fscreen": "^1.2.0",
-    "ipycanvas": "^0.8.2",
+    "ipycanvas": "^0.13.3",
     "jupyter-threejs": "^2.3.0"
   },
   "devDependencies": {
```

Comment:
these deps were copy/pasted from the current cookiecutter, which is configured for jupyterlab 4. I did remove the jest and babel deps that are in the cookiecutter, because we're not using them anywhere (and don't need to as far as I can tell)

Reply:
So there's a possibility that I added them when wasm-bundling wasn't as straightforward as it is now.